### PR TITLE
feat: create backup CI that doesn't use Earthly

### PR DESCRIPTION
**Pull request by skylar-simoncelli**

### CODE REVIEW

File: .github/workflows/not-earthly.yml [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+      - name: Build partner-chains-node
+        run: |
+          cargo build -p partner-chains-node --locked --release --target x86_64-unknown-linux-gnu
+          cp target/x86_64-unknown-linux-gnu/release/partner-chains-node partner-chains-node
+
+      - name: Transform YAML to JSON and Build chain specs
+        run: |
+          mkdir -p configs
+          yq eval-all -i '. as $item ireduce ({}; . * $item )' sidechains-infra-priv/src/kube/substrate-poc/environments/helm/substrate-node-stack-chart/values/chains/*.yaml -o=json
```
Comment (LGLO) [inline .github/workflows/not-earthly.yml:56]: Does Earthly use it as a source of truth?

Comment (LGLO) [reply .github/workflows/not-earthly.yml:56]: Earthfile uses values from this repo.
```
chainspecs:
  FROM +setup
  DO +INSTALL
  COPY devnet/.envrc devnet/.envrc
  COPY devnet/addresses.json devnet/addresses.json
  COPY staging/.envrc staging/.envrc
  COPY staging/addresses.json staging/addresses.json
```
Why this change?

Comment (skylar-simoncelli) [reply .github/workflows/not-earthly.yml:56]: It does not use this?

https://github.com/input-output-hk/sidechains-infra-priv/blob/master/Earthfile

Comment (skylar-simoncelli) [reply .github/workflows/not-earthly.yml:56]: Yes you are correct

Comment (LGLO) [reply .github/workflows/not-earthly.yml:56]: This is Earthlfile in infra-priv repo. We have problems with Earthfile in our repo. I don't see reference to sidechains-infra-priv in Earthfile nor in earthly.yml.

**Review by LGLO — APPROVED** [review]

**Pull request merged**